bundle/Form/Builder/FormBuilder.php

```diff
@@ -70,9 +62,16 @@ public function createUpdateFormForLocation(Location $location, Collection $coll
             );
     }
 
-    private function fromLegacyValue(FieldValue $legacyData, FieldDefinition $field)
+    private function fromLegacyValue(Attribute $attribute)
```

Comment:
This can be renamed to `fromAttribute` now :)

Reply:
this was fixed in 0b4f8204 commit